cli/coda.ts

```diff
@@ -34,6 +34,16 @@ if (require.main === module) {
           string: true,
           desc: 'For a dynamic sync table with a variable source location, specify the URL to test here.',
         },
+        timers: {
+          boolean: true,
+          default: false,
+          desc: 'Set to true to enable timer methods in the build. i.e. setTimeout',
+        },
+        browserifyWithEsbuild: {
```

Comment:
unfortunately I confirmed that esbuild is not worth the trust to browserify node modules. there are tons of issues to resolve "browser" dependencies that esbuild isn't correctly handling. for example https://github.com/evanw/esbuild/issues/772. 

I also tested with our packs repo. the local "browser" field mapping doesn't work well with multiple entries that depend on each other.